src/Common/src/CoreLib/System/MemoryExtensions.cs

```diff
@@ -245,12 +255,34 @@ public static bool SequenceEqual<T>(this Span<T> first, ReadOnlySpan<T> second)
             where T : IEquatable<T>
         {
             int length = first.Length;
-            if (typeof(T) == typeof(byte))
+            if (typeof(T) == typeof(byte) || typeof(T) == typeof(sbyte))
```

Comment:
The same change needs to be made for the ReadOnlySpan extension method:
https://github.com/dotnet/corefx/blob/7b9be0bb47f7e2d6f7824dc5fe4c77e8202e3028/src/Common/src/CoreLib/System/MemoryExtensions.cs#L634
```C#
public static bool SequenceEqual<T>(this ReadOnlySpan<T> first, ReadOnlySpan<T> second)
```

Reply:
Will do.